src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -60,37 +71,42 @@ public boolean doesUserExist(final String userName) {
 
   @Override
   public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
+    updateUser(user, hashedPassword, hashedPassword.isBcrypted() ? "bcrypt_password" : "password");
+  }
+
+  private void updateUser(final DBUser user, final HashedPassword hashedPassword, final String column) {
     Preconditions.checkArgument(user.isValid(), user.getValidationErrorMessage());
 
     try (final Connection con = connectionSupplier.get();
         final PreparedStatement ps =
-            con.prepareStatement("update ta_users set password = ?,  email = ?, admin = ? where username = ?")) {
+            con.prepareStatement("update ta_users set " + column + " = ?,  email = ? where username = ?")) {
       ps.setString(1, hashedPassword.value);
       ps.setString(2, user.getEmail());
-      ps.setInt(3, user.isAdmin() ? 1 : 0);
-      ps.setString(4, user.getName());
+      ps.setString(3, user.getName());
       ps.execute();
       con.commit();
     } catch (final SQLException e) {
-      throw new IllegalStateException(
-          "Error updating name:" + user.getName() + " email: " + user.getEmail() + " pwd: " + hashedPassword.mask(), e);
+      throw new IllegalStateException(String.format("Error updating name: %s email: %s pwd: %s",
+          user.getName(), user.getEmail(), hashedPassword.mask()), e);
     }
   }
 
   @Override
   public void createUser(final DBUser user, final HashedPassword hashedPassword) {
+    createUser(user, hashedPassword, hashedPassword.isBcrypted() ? "bcrypt_password" : "password");
```

Comment:
Part of the core change

Reply:
The changes are unrelated enough, different branches I think would have been better. 
- A couple of blockers are going to hold everything up, you may see some merge conflicts as a result of things dragging out
- It's hard to review this many independent changes. I have to figure out which updates are to support other changes and which ones are introducing new concepts.